### ISSUE TITLE
Shouldn't links that modify content be a POST request?

### ISSUE THREAD

**Issue by pilz** [ploneintranet/ploneintranet.prototype#97]
In the task management of cases we have links styled as buttons which issue a workflow change (Close/Reopen Milestone). This is a write to the data. It would be good if that would be a POST request. 

That would mean that the links should become submitting buttons of a form. 

@mattss raised that while reviewing the case branch and we thought we should bring that up for discussion with @cornae and see if we need to change the markup here.

**Comment by cornae** [comment]
The close milestone one? No I would say you a confirmation first that explains the consequences and there you have the submit button.

**Comment by pilz** [comment]
Smart solution. Same for reopen. Classic modal?

**Comment by cornae** [comment]
Yes sounds good. I guess the design stopped there during the München sprint and didn't think about it any more.
